crates/oxc_cli/src/options.rs

```diff
@@ -27,6 +28,10 @@ impl<'a> TryFrom<&'a ArgMatches> for CliOptions {
             paths.extend(globbed);
         }
 
-        Ok(Self { quiet: matches.get_flag("quiet"), paths })
+        Ok(Self {
+            quiet: matches.get_flag("quiet"),
+            max_warnings: matches.get_one::<isize>("max-warnings").copied().unwrap_or(-1),
```

Comment:
Just trying to match the behavior outlined in the ESLint docs here but defaulting to -1

Reply:
I'd go for `None` because we are writing Rust :-)

Reply:
The clippy warning will be gone after changing to `None` and `usize.

Reply:
Sounds good, thanks.